executor/adapter.go

```diff
@@ -213,6 +214,7 @@ func (a *ExecStmt) PointGet(ctx context.Context, is infoschema.InfoSchema) (*rec
 		defer span1.Finish()
 		ctx = opentracing.ContextWithSpan(ctx, span1)
 	}
+	ctx = a.setPProfLabelWithPlan(ctx)
```

Comment:
1. There is only 2 place need to call this function, no need to call in every executor. Since the `point-get` with cache plan is a special handle, we should also do special handle it for Top-SQL too. Currently, there is no other place need to call this function.

2.  this is called multiple times doesn't affect the correctness, so It's idempotent.